store/tikv/coprocessor.go

```diff
@@ -638,6 +685,14 @@ func (it *copIterator) Next(ctx context.Context) (kv.ResultSubset, error) {
 		ok     bool
 		closed bool
 	)
+
+	// broadcast should be called under it.actionOnExceed.cond critical area
+	broadcast := func() {
+		it.actionOnExceed.cond.exceed = false
```

Comment:
should we reset `teared` here

Reply:
It's ok to reset `teared` here while it has already been reset in `Action`

Reply:
we can define a member function `broadcast` for action?
and if possible, we can reuse this func in Close